src/lib/datepicker/calendar.html

```diff
@@ -15,7 +15,7 @@
   </div>
 </div>
 
-<div class="mat-calendar-content" tabindex="0" (keydown)="_handleCalendarBodyKeydown($event)"
+<div class="mat-calendar-content" tabindex="-1" (keydown)="_handleCalendarBodyKeydown($event)"
```

Comment:
It was actually immediately shifting its focus to one of the `gridcell`, so it was never focused for long enough to be read. I refactored to cut out the middleman though, and just directly focus the `gridcell` now